src/lang/lexer.ts

```diff
@@ -14,8 +14,8 @@ export class Lexer {
     private buffer: string = "";
     constructor(source: string) {
         this.source = source
-            .replace("<?php", "@php")
-            .replace("?>", "@endphp")
+            .replace(/<\?=\s+(\".+\")\s+\?\>/, "{{!! $1 !!}}")
```

Comment:
```suggestion
            .replace(/<\?=\s+(\".+\")\s+\?\>/, "{!! $1 !!}")
```
https://laravel.com/docs/master/blade#displaying-unescaped-data

Reply:
Yep, just saw, the reason the test is not passing is because the test is wrong also 😆